tests/jobs.py

```diff
@@ -270,21 +269,21 @@ def test_dagrun_root_fail(self):
             },
             dagrun_state=State.FAILED)
 
-    def test_dagrun_deadlock(self):
-        """
-        Deadlocked DagRun is marked a failure
-
-        Test that a deadlocked dagrun is marked as a failure by having
-        depends_on_past and an execution_date after the start_date
-        """
-        self.evaluate_dagrun(
-            dag_id='test_dagrun_states_deadlock',
-            expected_task_states={
-                'test_depends_on_past': None,
-                'test_depends_on_past_2': None,
-            },
-            dagrun_state=State.FAILED,
-            advance_execution_date=True)
+    # def test_dagrun_deadlock(self):
```

Comment:
After I made this change

https://github.com/apache/incubator-airflow/pull/1636/files#diff-a32a363fa616685db3bfefba947535b2R3712

I noticed that the logic for detecting a deadlocked DAG doesn't apply if any of the tasks have `depends_on_past` set to true. Since this test uses a DAG with `depends_on_past` set to true, it doesn't seem to be applicable anymore.